ramp-frontend/ramp_frontend/tests/test_ramp.py

```diff
@@ -385,6 +386,34 @@ def test_ask_for_event_mail(client_session):
                         in outbox[0].body)
 
 
+@pytest.mark.parametrize(
+    "opening_date,public_date,closing_date,expected", testtimestamps
+)
+def test_submit_button_enabled(client_session, makedrop_event,
+                               opening_date, public_date,
+                               closing_date, expected):
+    client, session = client_session
+
+    event = get_event(session, 'iris_test_4event')
+    event.opening_timestamp = opening_date
+    event.public_opening_timestamp = public_date
+    event.closing_timestamp = closing_date
+    session.commit()
+    sign_up_team(session, 'iris_test_4event', 'test_user')
+
+    with login_scope(client, 'test_user', 'test') as client:
+        event_info = {
+            'button': 'Submit'
+        }
+        rv = client.get('http://localhost/events/iris_test_4event/sandbox',
+                        data=event_info)
+        assert rv.status_code == 200
+        if expected == b'event-close':
```

Comment:
Put a small comment explaining that the button should be disable and it will appear in the html data. 

The reason is that the test is a bit loose since more element could be disabled but this is enough here. However in case of a change in the future it will be difficult to remember what is disabled at a first glance.